tests/Propel/Tests/Generator/Builder/Om/QueryBuilderTest.php

```diff
@@ -1389,4 +1390,44 @@ public function testPruneCompositeKey()
         $nbBookListRel = BookListRelQuery::create()->prune($testBookListRel)->count();
         $this->assertEquals(1, $nbBookListRel, 'prune() removes an object from the result');
     }
+
+    /**
+     * @return void
+     */
+    public function testFindPkSimpleIsAbstractWhenTableIsAbstract(): void
+    {
+        $databaseXml = '
+<database>
+    <table name="my_table" abstract="true">
+        <column name="id" type="integer" primaryKey="true" />
```

Comment:
Let's use the consistent version without trailing space.
```suggestion
        <column name="id" type="integer" primaryKey="true"/>
```

Reply:
Ok